### DISCUSSION TITLE
Error running ios app on ios 17 simulator

### DISCUSSION THREAD

**Discussion by jacobc94** [AvaloniaUI/Avalonia#15825] [Q&A] [answered]
I created a simple app in avalonia. It runs fine on Android and Desktop but when running on an iOS 17 simulator it gives me an error message: `Microsoft.iOS: Socket error while connecting to IDE on 127.0.0.1` and crashes with a double fault: 
Got a SIGBUS while executing native code. This usually indicates a fatal error in the mono runtime or one of the native libraries used by your application.
I can get it running on an ios 16 simulator but asking customers to downgrade is not an option. 
I have tried updating Net to 8.0.300 as well... any help?

**Comment by jp2masa** [comment, marked as answer]
Duplicate of #15135, #15778?

**Comment by dcwang-lab** [comment]
解决了吗，我也曾遇到过这种问题，可以发我邮箱2099961945@qq.com 看能否帮您解决。

**Reply by jacobc94** [reply]
Nope. It seems like it's a Microsoft issue

**Comment by suchithm** [comment]
Microsoft.iOS: Socket error while connecting to IDE on 127.0.0.1
In my case i was clicking run(play) button in rider instead if i select debug it is working.
